api_specifications/case-api-0.1.0.yaml

```
@@ -459,6 +459,61 @@ paths:
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/JoinCaseV2Response'
+        '400':
+          description: Failed to verify the given litigants and representatives to be added to the case
+          content:
+            '*/*':
+              schema:
+                $ref: >-
+                  https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)